tasks/build/config/kibana.yml

```diff
@@ -0,0 +1,104 @@
+# Kibana is served by a back end server. This setting specifies the port to use.
+#server.port: 5601
```

Comment:
Why do we need a copy of the kibana.yml file? Can't we copy it from `config/kibana.yml` at build time?

Reply:
yep, there's a one line diff on this one with the logging directory set.  the only reason i copied is because i figure it's less likely to break on accident but i'm indifferent.

i can get a replace with lots of error checking setup.

Reply:
hmm, maybe we should make the kibana.yml file a template that can produce both rather than committing the final kibana.yml file to the repo